client/types/charges.d.ts

```diff
@@ -1,8 +1,54 @@
+/**
+ * Internal dependencies
+ */
+import { BalanceTransaction } from './balance_transactions';
+import { Dispute } from './disputes';
+
 interface BillingDetails {
 	name: string;
 }
 
+interface ChargeOutcome {
+	network_status: string;
+	reason: string;
+	risk_level: string;
+	risk_score: number;
+	rule: string;
+	seller_message: string;
+	type: string;
+}
+
+interface ChargeRefund {
+	balance_transaction: BalanceTransaction;
+}
+
+interface ChargeRefunds {
+	data: ChargeRefund[];
+}
+
 export interface Charge {
 	id: string;
+	amount: number;
+	amount_captured: number;
+	amount_refunded: number;
+	application_fee_amount: number;
+	balance_transaction: BalanceTransaction;
 	billing_details: BillingDetails;
+	captured: boolean;
+	currency: string;
+	dispute?: Dispute;
```

Comment:
I haven't tested it but would this result in a circular reference when Karlis' changes are merged?  https://github.com/Automattic/woocommerce-payments/pull/3491/files#diff-3281997a33f70ad0c3013fdb32857f43264b59ddf3b76dd1f3aef2d0b8b435ed

Reply:
I'd suspect that not all related object are included in the response from API in both cases. So accroding to what is returned, here would be a good place for `Omit` usage?

Reply:
I'm waiting to cross that bridge when we get to it 🙂 Honestly, I have no idea what would happen.

Reply:
Actually, in my branch there is already a circular dependency `Charge` <-> `Dispute`. Fortunately, TypeScript pre-processor can handle it.